pkg/encoding/utils/pointsIO.go

```diff
@@ -30,19 +47,19 @@ func ReadG1Points(filepath string, n uint64, numWorker uint64) ([]bls.G1Point, e
 	}()
 
 	startTimer := time.Now()
-	g1r := bufio.NewReaderSize(g1f, int(n*64))
+	g1r := bufio.NewReaderSize(g1f, int(n*G1PointBytes))
 
 	if n < numWorker {
 		numWorker = n
 	}
 
-	buf, _, err := g1r.ReadLine()
+	buf, err := ReadFile(g1r, n*G1PointBytes)
 	if err != nil {
 		return nil, err
 	}
 
-	if uint64(len(buf)) < 64*n {
-		log.Printf("Error. Insufficient G1 points. Only contains %v. Requesting %v\n", len(buf)/64, n)
+	if uint64(len(buf)) < G1PointBytes*n {
```

Comment:
cool. it is checked here

Reply:
wait not, the buf capacity is created to be equal

Reply:
Yea this will always false given the impl of `ReadDesiredBytes`. Cleaned up!